src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -441,7 +446,7 @@ const useGroupInfo = (channelId = null) => {
 
     if (isAnonymousUser) {
       const modifiedUser = {...item};
-      modifiedUser.user.anonymousUsername = `Anonymous ${item?.user?.anon_user_info_emoji_name}`;
+      modifiedUser.user.anonymousUsername = `${item?.user?.anon_user_info_color_name} ${item?.user?.anon_user_info_emoji_name}`;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the assignment of `anonymousUsername` is a good change as it now uses the `anon_user_info_color_name` along with `anon_user_info_emoji_name` to set the anonymous username.
```diff
- modifiedUser.user.anonymousUsername = `Anonymous ${item?.user?.anon_user_info_emoji_name}`;
+ modifiedUser.user.anonymousUsername = `${item?.user?.anon_user_info_color_name} ${item?.user?.anon_user_info_emoji_name}`;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->